common/runtime.hpp

```diff
@@ -15,8 +15,18 @@ namespace ILLIXR {
 		virtual void load_so(const std::vector<std::string>& so) = 0;
 		virtual void load_so(const std::string_view so) = 0;
 		virtual void load_plugin_factory(plugin_factory plugin) = 0;
+
+		/**
+		 * Returns when the runtime is completely stopped.
+		 */
 		virtual void wait() = 0;
+
+		/**
+		 * Requests that the runtime is completely stopped.
+		 * Clients must call this before deleting the runtime.
+		 */
 		virtual void stop() = 0;
+
 		virtual ~runtime() {}
```

Comment:
```suggestion
		virtual ~runtime() = default;
```
I think this is slightly different semantically than an empty body (`{ }`), but I'm still not sure which should be [preferred](https://stackoverflow.com/questions/13576055/how-is-default-different-from-for-default-constructor-and-destructor). I've been switching to `= default` as I edit.

Reply:
I think `default` is preferred so that the destructor is considered "trivial." It should be trivial unless we have a reason to make it non-trivial.